metaseq/cli/train.py

```diff
@@ -521,6 +521,9 @@ def post_checkpoint_callback(cfg, num_updates, training_finished, filename):
                 f"Successfully copied {filename} to {cfg.checkpoint.cloud_upload_path}"
             )
             os.remove(filename)
+
+            # TODO[Susan]: Add symlink logic here? Check what cloud_upload_path is being used for Uriel's jobs.
```

Comment:
we started using cloud_upload_path but there was an issue sometimes. maybe its due to the checkpoint_last issue here, but anyway we stopped using it and went back to local filesystem..

Reply:
in line 476 in this file we use `async_callback_fn` only if `cfg.checkpoint.cloud_upload_path==True`, but it should work for local storage as well, no?